src/Handler.php

```diff
@@ -69,15 +137,23 @@ public function exceptionHandler($throwable)
      */
     public function errorHandler($errno, $errstr, $errfile = '', $errline = 0)
     {
-        if ($this->client->shouldIgnoreErrorCode($errno)) {
-            return;
-        }
```

Comment:
FWIW, the interpreter interprets this `null` return value differently than the `false` you were returning below.

> It is important to remember that the standard PHP error handler is completely bypassed for the error types specified by error_types unless the callback function returns FALSE.
>
> http://php.net/manual/en/function.set-error-handler.php

We can verify this:

```
$ cat test-exits-null.php
<?php

set_error_handler(
    function ($errno, $errstr) {
        return;
    }
);

trigger_error("Uh oh!", E_USER_WARNING);
$ php test-exits-null.php
$ # warning not outputted
```

```
$ cat test-exits-false.php
<?php

set_error_handler(
    function ($errno, $errstr) {
        return false;
    }
);

trigger_error("Uh oh!", E_USER_WARNING);
$ php test-exits-false.php
PHP Warning:  Uh oh! in /private/tmp/test-exits-false.php on line 9

Warning: Uh oh! in /private/tmp/test-exits-false.php on line 9
```